pkgs/development/tools/rust/bindgen/unwrapped.nix

```diff
@@ -7,18 +7,15 @@ let
   rustfmt-nightly = rustfmt.override { asNightly = true; };
 in rustPlatform.buildRustPackage rec {
   pname = "rust-bindgen-unwrapped";
-  version = "0.59.2";
+  version = "0.61.0";
 
-  RUSTFLAGS = "--cap-lints warn"; # probably OK to remove after update
-
-  src = fetchFromGitHub {
-    owner = "rust-lang";
-    repo = "rust-bindgen";
-    rev = "v${version}";
-    sha256 = "sha256-bJYdyf5uZgWe7fQ80/3QsRV0qyExYn6P9UET3tzwPFs=";
+  src = fetchCrate {
+    pname = "bindgen-cli";
+    inherit version;
+    sha256 = "sha256-sKcKIAkUC2GfAZ4tJBNweXhoFzqO95iCpHgekpOyHzc=";
```

Comment:
Sadly fetching from GitHub results in `error: the lock file /build/source/Cargo.lock needs to be updated but --frozen was passed to prevent this`.  I used `fetchCrate` to avoid patching the `Cargo.lock`.

There are no prior releases for `bindgen-cli` on crates.io because the CLI was separated from the library in this release.  Future releases will be under the `bindgen-cli` crate.

Do you still want me to add a comment, and if so explaining what exactly? Sorry to be a little dense, but I want to make sure I understand what it is that should be documented :)

Reply:
> There are no prior releases for `bindgen-cli` on crates.io because the CLI was separated from the library in this release. Future releases will be under the `bindgen-cli` crate.

Thanks for the clarification.  In that case I think this is fine.